Exec/hydro_tests/Sedov/inputs.mini-Castro

```diff
@@ -44,6 +44,10 @@ amr.refine.dengrad_rel.max_level = 20
 amr.refine.dengrad_rel.relative_gradient = 0.25
 amr.refine.dengrad_rel.field_name = density
 
+amr.refine.dengrad_rel.max_level = 20
+amr.refine.dengrad_rel.relative_gradient = 0.25
+amr.refine.dengrad_rel.field_name = Temp
```

Comment:
I think what you want here is to add a `tempgrad_rel` to `refinement_indicators` and update the name for these lines appropriately (or replace line 45 with `Temp` instead of `density` and rename all of the `dengrad_rel` to `tempgrad_rel`).

Reply:
updated